_plugins/build.rb

```diff
@@ -0,0 +1,12 @@
+Jekyll::Hooks.register :site, :pre_render do |site|
+    # TODO: remove 'js-optimization'
+    production_branches = ['master', 'staging', 'js-optimization']
+    branch = ENV['BRANCH']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.